pages/playlists/index.tsx

```diff
@@ -17,6 +17,18 @@ const Playlist: NextPage = () => {
     isLoadingMore,
   } = usePaginatedPlaylists();
 
+  if (!playlists) {
```

Comment:
So, doing this actually removes the entire DashboardLayout. Now looking, we should add this to the return function below rather than returning two separate layouts. Down below you can add the following:

```
{!playlists && (
  // empty state here
)}
```